causalml/inference/nn/cevae.py

```diff
@@ -25,10 +25,14 @@
 import logging
 import torch
 from pyro.contrib.cevae import CEVAE as CEVAEModel
+
+from causalml.inference.meta.base import BaseLearner
 from causalml.inference.meta.utils import convert_pd_to_np
 
-logging.getLogger("pyro").setLevel(logging.DEBUG)
-logging.getLogger("pyro").handlers[0].setLevel(logging.DEBUG)
+pyro_logger = logging.getLogger("pyro")
+pyro_logger.setLevel(logging.DEBUG)
+if pyro_logger.handlers:
+    pyro_logger.handlers[0].setLevel(logging.DEBUG)
 
 
 class CEVAE(object):
```

Comment:
Should we inherit the `BaseLearner` here?

Reply:
We can't because currently `CEVAE` and `DragonNet` don't have `estimate_ate()`. I will remove it from `DragonNet` as well.

Reply:
Done.